drivers/hmis/app/graphql/mutations/update_project.rb

```diff
@@ -20,17 +20,21 @@ def resolve(id:, input:, confirmed:)
     end
 
     def create_errors(project, input)
-      return [] unless project.operating_end_date.blank? && input.operating_end_date.present?
-
       errors = HmisErrors::Errors.new
 
-      funder_count = project.funders.where(end_date: nil).count
-      inventory_count = project.inventories.where(inventory_end_date: nil).count
-      open_enrollments = Hmis::Hud::Enrollment.open_on_date.in_project_including_wip(project.id, project.project_id)
-
-      errors.add :base, :information, severity: :warning, full_message: "Project has #{open_enrollments.count} open #{'enrollment'.pluralize(open_enrollments.count)}." if open_enrollments.present?
-      errors.add :base, :information, severity: :warning, full_message: "#{funder_count} open #{'funder'.pluralize(funder_count)} will be closed." if funder_count.positive?
-      errors.add :base, :information, severity: :warning, full_message: "#{inventory_count} open inventory #{'record'.pluralize(inventory_count)} will be closed." if inventory_count.positive?
+      # If project end date is changing
+      if input.operating_end_date.present? && project.operating_end_date != input.operating_end_date
+        open_enrollments = Hmis::Hud::Enrollment.open_on_date(input.operating_end_date).in_project_including_wip(project.id, project.project_id)
+        errors.add :base, :information, severity: :warning, full_message: "Project has #{open_enrollments.count} open #{'enrollment'.pluralize(open_enrollments.count)} on the selected end date." if open_enrollments.present?
+      end
+
+      # If project is being "closed" for the first time
+      if project.operating_end_date.blank? && input.operating_end_date.present?
```

Comment:
I think we can check these too, the same restrictions should apply if you are changing the operating end date than if you are adding it.

Reply:
Oh, never mind, this is just checking if they are still open, not if the dates make sense.